apps/console/src/features/authentication/pages/sign-in.tsx

```diff
@@ -44,14 +46,14 @@ const SignIn: FunctionComponent<RouteComponentProps> = (
     const loginSuccessRedirect = () => {
 
         // TODO: Fix the auth callback isssue here.
```

Comment:
If this fixes the issue, shall we remove the TODO.

Reply:
Resolved by 32527d6